docs/RELEASE_PROCESS.md

```diff
@@ -0,0 +1,25 @@
+# Release process and schedule
+
+The release manager is responsible for ensuring that the following steps are performed:
+1. [ ] Fix the following dates (at least 10 weeks before the release, i.e. 5 weeks before the soft freeze):
+       soft freeze &rarr; (4 weeks) &rarr; hard freeze &rarr; (1 week) &rarr; release day
+1. [ ] As soon as they are fixed, communicate dates for soft freeze, hard freeze and release (via Github discussions).
+1. [ ] Go through pull requests and issues without a milestone and assign them to the upcoming release if necessary.
```

Comment:
We should clarify when this happens. At the next community meeting after setting the release date? Or should this part of fixing the release date?